text_editor.py

```diff
@@ -1,35 +1,90 @@
 """Text Editor"""
 
-# import keyboard
+# import curses
 
 class Model:
     """
     The model for the text editor.
```

Comment:
I think you need to write a short doc explaining the data structure, then you can paste it in here.